dct_backend/reviews.py

```diff
@@ -0,0 +1,62 @@
+from fastapi import FastAPI, Response, status
+from dotenv import load_dotenv
+import os
+from database import Database
+import server
+from fastapi.testclient import TestClient
+
+load_dotenv()
+db = Database(os.getenv("DB_USERNAME"), os.getenv("DB_PASSWORD"))
+db.connect()
+
+food_db = db.client["food"] # from food collection
+review_collection = food_db["reviews"] # review --> sub collection
+app = FastAPI()
```

Comment:
I think there's some confusion about the Fast API code. #105 does a lot of restructuring of it anyways so you may want to merge your branch with that one after the tests are fixed for it, but to get an overall understanding: the server will set the fast api code and create the `app`. You want to create a new route in `server.py` that uses the methods you wrote here. What you've done in this file is essentially created 2 servers. 1 in `server.py` and 1 in `reviews.py`. So technically you could run `uvicorn server:app` and `uvicorn reviews:app`, which is not what we want. I am not sure if that makes sense but I am happy to clarify more

Reply:
thanks for clarifying @RafaelPiloto10, so how do I make a new route in server.py? Do I use **router = APIRouter(prefix="/reviews")** ?

Reply:
Yes, @MimiOlayeye. That creates part of the route. That define that the `<our app>/reviews/<SOMETHING>` URL exists. You need to then add endpoints that will replace the `<SOMETHING>`. For example, `/add-review` or `/edit-review`. Here you would define the functionality that should be executed when we make requests to `https://<ourapp>/reviews/add-review`. Does that help? We can spend some time Saturday morning as well if you have any questions